features/import.feature

```diff
@@ -14,3 +14,41 @@ Feature: cli-kintone import command
       | Alice  | 10     |
       | Bob    | 20     |
       | Jenny  | 30     |
+
+  Scenario: CliKintoneTest-20 Should return the error message when there is no privilege to add records
+    Given The csv file "tmp/CliKintoneTest-20.csv" with content as below:
```

Comment:
Merging the latest main will apply https://github.com/kintone/cli-kintone/pull/553 , and you don't need to create a `tmp/` directory anymore.

```suggestion
    Given The csv file "CliKintoneTest-20.csv" with content as below:
```